packages/dashboard-frontend/src/services/bootstrap/index.ts

```diff
@@ -267,4 +271,23 @@ export default class Bootstrap {
     const { requestUserProfile } = UserProfileStore.actionCreators;
     return requestUserProfile()(this.store.dispatch, this.store.getState, undefined);
   }
+
+  private async checkWorkspaceStopped() {
+    const path = SessionStorageService.remove(SessionStorageKey.ORIGINAL_LOCATION_PATH);
+    if (!path || path === '/') {
+      return;
+    }
+
+    const state = this.store.getState();
+    if (state.devWorkspaces.workspaces.some(w => w.status?.mainUrl?.includes(path))) {
+      this.issuesReporterService.registerIssue(
+        'workspaceStopped',
+        new Error(
+          'Your workspace has stopped. This could happen if your workspace shuts down due to inactivity.',
+        ),
+      );
+    }
+
+    await new Promise(resolve => setTimeout(resolve, 10000));
```

Comment:
Why do we need this timeout?

```
await new Promise(resolve => setTimeout(resolve, 10000));
```

I think it could be just

```
private checkWorkspaceStopped(): void {
...
```

Reply:
@dkwon17 was this 1sec delay added by purpose?